earth_enterprise/src/fusion/autoingest/JsonUtils_unittest.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright 2018 the Open GEE Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// Unit tests for Js Utilities 
+
+#include <string>
+#include <gtest/gtest.h>
+#include "fusion/autoingest/JsonUtils.h"
+
+namespace {
+
+class JsonUtilsAccessor : public JsonUtils {
+  public:
+    std::string RunJsonBool(bool value) {
+      return JsonBool(value);
+    }
+  
+    std::string RunLookAtJson(const std::string& lookAtSpec){
+      return LookAtJson(lookAtSpec);
+    }
+
+    std::string RunJsonObject(const std::map<std::string, std::string>& field_map){
+      return JsonObject(field_map);
+    }
+
+    std::string RunJsonArray(const std::vector<std::string>& array_values){
+      return JsonArray(array_values);
+    }
+};
+
+
+class JsonUnitTest : public testing::Test, public JsonUtils {
+  public:
+    JsonUtilsAccessor jsonUtils;
+};
+
+// Testing the JsonBool function
+TEST_F(JsonUnitTest, JsonBoolTrue) {
+    const bool trueBool = true;
+    const std::string expectedTrue= "true";
+    std::string result = jsonUtils.RunJsonBool(trueBool);
+
+    EXPECT_EQ(expectedTrue, result);
+}
+
+TEST_F(JsonUnitTest, JsonBoolFalse) {
+    const bool falseBool = false;
+    const std::string expectedFalse= "false";
+    std::string result = jsonUtils.RunJsonBool(falseBool);
+
+    EXPECT_EQ(expectedFalse, result);
+}
+
+TEST_F(JsonUnitTest, LookAtValid) {
+    const std::string lookAtString = "-93.02119299429238|38.88758732203867|4963567.527932385|0.4781091614682265|6.862074527538373";
+    const std::string expectedJson= "{\naltitude : 4.96357e+06,\nlat : 38.88758732203867,\nlng : -93.02119299429238,\nzoom : 3\n}\n";
+    std::string result = jsonUtils.RunLookAtJson(lookAtString);
+
+    EXPECT_EQ(expectedJson, result);
+}
+
+TEST_F(JsonUnitTest, LookAtInvalid) {

```

Comment:
Minor: you could also add a test for LookAt with an empty string. That will increase coverage.